src/Domain/FunctionsOfTime/IntegratedFunctionOfTime.cpp

```diff
@@ -0,0 +1,143 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/FunctionsOfTime/IntegratedFunctionOfTime.hpp"
+
+#include <algorithm>
+#include <iterator>
+#include <memory>
+#include <ostream>
+
+#include "DataStructures/DataVector.hpp"
+#include "Domain/FunctionsOfTime/ThreadsafeList.tpp"
+#include "Utilities/ErrorHandling/Error.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace domain::FunctionsOfTime {
+
+IntegratedFunctionOfTime::IntegratedFunctionOfTime() = default;
+IntegratedFunctionOfTime::IntegratedFunctionOfTime(IntegratedFunctionOfTime&&) =
```

Comment:
Can these just go in the hpp? Or did clang-tidy yell at you?

Reply:
They need the tpp definitions visible.